spring-core/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.togglz</groupId>
     <artifactId>togglz-project</artifactId>
-    <version>3.2.1-SNAPSHOT</version>
+    <version>4.0.0-SNAPSHOT</version>
```

Comment:
`spring-core` module still contains `spring,factories` file not converted to `org.springframework.boot.autoconfigure.AutoConfiguration.imports`

Reply:
According to https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/test/context/TestExecutionListener.html the spring.factories file is still the way to go for TestExecutionListener registration in Spring 6.

Reply:
It's also still mentioned here: https://docs.spring.io/spring-framework/docs/current/reference/html/testing.html#testcontext-tel-config-automatic-discovery

There's also a statement on https://stackoverflow.com/questions/73317916/spring-boot-2-7-migrating-spring-factories-into-imports-file which states that spring.factories is deprecated for auto-configuration classes and test slices, but not for registering TestExecutionListener instances for the tests.